test/v1/models/errors/ErrorsSpec.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.errors
+
+import play.api.libs.json.Json
+import support.UnitSpec
+
+class ErrorsSpec extends UnitSpec {
```

Comment:
As stated above, it would be best to take safety precautions against potentially empty error case classes.